docs/book/src/plugins/extending/external-plugins.md

```diff
@@ -148,4 +148,4 @@ kubebuilder create api --plugins go/v4,sampleplugin/v1
 - A [sample external plugin written in Python](https://github.com/rashmigottipati/POC-Phase2-Plugins)
 - A [sample external plugin written in JavaScript](https://github.com/Eileen-Yu/kb-js-plugin)
 
-[code-plugin-external]: ./../../../../../pkg/plugin/external/types.go
+[code-plugin-external]: https://github.com/kubernetes-sigs/kubebuilder/blob/book-v4/pkg/plugin/external/types.go
```

Comment:
Could we change to use the right relative paths?

Reply:
Hmm, I think the issue here is that we are not pointing to a .md file.
But I can take a look in the mdBook docs to see if it's possible.

Reply:
I see that makes sense 
we have a way to add code 
I would need to give a look, but we are probably not using the core right here
See in other places that we can inject the code on the docs via annotations